app/knight.py

```diff
@@ -0,0 +1,53 @@
+class Knight:
+    def __init__(self, config: dict) -> None:
+        self.name = config["name"]
```

Comment:
Ensure that the 'name' key exists in the config dictionary to avoid KeyError. Consider using `config.get('name', default_value)` to provide a default value if the key is missing.